protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,80 @@
+# Deputy Pause Module
+
+## Context
+
+The Foundation Safe is given the ability to act as the Security Council Safe for a limited number
+of safety-net actions on OP Mainnet. For instance, the Foundation Safe can act as the Security
+Council Safe to trigger the Superchain-wide pause function. These powers are given to the
+Foundation Safe through the `DeputyGuardianModule` installed in the Security Council Safe.
+
+## Problem Statement
+
+The Foundation Safe pre-signs certain transactions to reduce response time in case of an emergency.
+Because pre-signed Safe transactions are dependent on the nonces of that Safe, these pre-signed
+transactions become invalidated any time the Foundation Safe is required to sign literally any
+other transaction. In an attempt to reduce the overhead from this state of affairs, a *second*
+Foundation Safe sometimes referred to as the "Foundation Operations Safe" with the exact same
+configuration as the original Foundation Safe was made to be the Deputy Guardian.
+
+Even with this second Foundation Safe, pre-signed pauses are invalidated on a regular basis
+whenever an upgrade touches the `DeputyGuardianModule`. Gripes with the pre-signed pause system
+could fill a whole role of toilet paper and are not just limited to the issues noted above.
+
+## Proposed Solution
+
+We propose eliminating the pre-signed pause system entirely by introducing a `DeputyPauseModule` to
+be installed into the primary Foundation Safe. The `DeputyPauseModule` would allow a deputy account
+to act as the Foundation Safe to trigger the Superchain-wide pause functionality. The deputy
+account would not have any other capabilities.
```

Comment:
Let's get a bit more specific on the Safe changes. Currently we have:
- `0x847B5c174615B1B7fDF770882256e2D3E95b9D92`: Known as "Foundation Upgrades Safe", deployed on 2024-02-08, this is used as one owner on the 2/2 upgrader (L1 ProxyAdmin Owner) Safe with the security council.
- `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`: Known as "Foundation Operations Safe", deployed on 2021-01-16, this is used for presigned pauses, and is also one signer on Base's 2/2 Safe

Because 0x9B is older, I assume this is the primary foundation safe being referred to? So we'd install the module on this 0x9B safe, change the 2/2 with the security council to use this Safe, and deprecate the 0x84 safe? We should ask the data team to help find any usage of that safe that we might be forgetting (i.e. any contract with `847B5c174615B1B7fDF770882256e2D3E95b9D92`, or it's alias, somewhere in storage)